Library/Formula/pxz.rb

```diff
@@ -34,3 +34,16 @@ def install
  		if (m == MAP_FAILED) {
  			perror("mmap failed");
  			exit(EXIT_FAILURE);
+diff --git a/Makefile.orig b/Makefile
```

Comment:
This is already fixed in upstream HEAD: https://github.com/jnovy/pxz/blob/master/Makefile#L15

The maintainer is dragging his feet making a fresh release though: https://github.com/jnovy/pxz/issues/21